<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatRoomList</class>
 <widget class="QWidget" name="ChatRoomList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>채팅방 목록</string>
  </property>
  <property name="styleSheet">
   <string>
    QWidget {
        background-color: #f0f8ff;
        font-family: Arial, sans-serif;
    }
    QPushButton {
        background-color: #87ceeb;
        border: 1px solid #4682b4;
        border-radius: 8px;
        color: white;
        font-weight: bold;
        padding: 5px 10px;
    }
    QPushButton:hover {
        background-color: #4682b4;
    }
    QListWidget, QTableWidget {
        border: 1px solid #87ceeb;
        border-radius: 5px;
        padding: 5px;
    }
    QLabel {
        font-size: 14px;
        font-weight: bold;
        color: #4682b4;
    }
   </string>
  </property>
  <widget class="QLabel" name="headerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>동행 찾기 플랫폼 MeEither에 오신 것을 환영합니다!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="chatRoomListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>79</y>
     <width>400</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="createRoomButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>채팅방 만들기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinRoomButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>470</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>채팅방 들어가기</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="userInfoTable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>201</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>
     QTableWidget {
         background-color: #f9f9f9;
         border: 1px solid #87ceeb;
         border-radius: 5px;
         padding: 5px;
     }
    </string>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <property name="horizontalHeaderLabels" stdset="0">
    <stringlist>
     <string>닉네임</string>
     <string>나이</string>
     <string>성별</string>
    </stringlist>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>60</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>17</number>
   </attribute>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="viewProfileButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>프로필 보기</string>
   </property>
  </widget>
  <widget class="QListWidget" name="state">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="continentLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>대륙</string>
   </property>
  </widget>
  <widget class="QLabel" name="myInformation">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>접속자</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
